databrowser/src/config/tourism/webcamInfo/webcamInfo.listView.ts

```diff
@@ -0,0 +1,63 @@
+import { ListViewConfig } from '../../../domain/datasetConfig/types';
+import { CellComponent } from '../../../domain/cellComponents/types';
+
+export const eventTopicsListView: ListViewConfig = {
```

Comment:
Variable name seems wrong - should probably be something like "webcamInfoListView"